.github/actions/build_and_push_docker_image/action.yml

```diff
@@ -46,12 +46,6 @@ runs:
     - name: Build docker image 🐳
       run: docker build -t ${DOCKERHUB_ORGANISATION}/${APP_NAME}:${APP_VERSION} -t ${DOCKERHUB_ORGANISATION}/${APP_NAME}:${{ github.ref_name }} .
       shell: bash
-    - name: Verify nginx image
-      run: |
-        set -e
-        docker run --rm ${DOCKERHUB_ORGANISATION}/${APP_NAME}:${{ github.ref_name }} nginx -t
-        echo "docker image validated successfully"
-      shell: bash
```

Comment:
We don't want to validate the nginx config anymore?

Reply:
at least for now, this thing doesnt work. It needs to be investigated more.